ingestionScripts/metadata_ingestion_job.kjb

```diff
@@ -386,12 +386,12 @@
         </field>
         <field>
           <variable_name>packageDirectory</variable_name>
-          <variable_value>MetadataTesting2</variable_value>
+          <variable_value>KPMP-1628</variable_value>
```

Comment:
Will this be overwritten with new values if I run it?

Reply:
This variable has to be changed every time it runs so it doesn't really matter what it's set to. Not sure why this changed but it wasn't really part of this card.